app/validators/routes_and_redirects_validator.rb

```diff
@@ -123,7 +123,6 @@ def initialize(record, redirect)
     def validate
       path = redirect[:path]
       destination = redirect[:destination]
-      type = redirect[:type]
```

Comment:
It's not something that has become unused through other changes made here, so I've moved it to be the earliest commit in the branch.